cypress/e2e/common/cmsSteps.js

```diff
@@ -183,3 +183,103 @@ And('{string} date stamp displays todays date', (stampLbl) => {
 });
+
+And('user clicks on {string} sub tab', (contentSubTab) => {
+    cy.get(`li.tabs__tab a:contains("${contentSubTab}")`).click({ force: true });
+});
+
+And('user uploads test file {string}', (fileName) => {
+    cy.fixture(fileName, { encoding: null }).as('fixture')
+    cy.get('input[type=file]').selectFile('@fixture');
+    cy.wait(2000);
+});
+
+And('user clicks on dropdown button toggle to view all Related Resources types', () => {
+    cy.get('.dropbutton-toggle button').click({ force: true });
+})
+And('user selects {string} related resource item', (resourceType) => {
+    cy.get(`input[value='${resourceType}']`).click({ force: true });
+});
+Then('{string} section appears', (resourceType) => {
+    cy.get(`tbody div:contains("${resourceType}")`).should('be.visible');
+})
+And('user clicks on {string} button to link to a resource', (link) => {
+    cy.get(`tbody summary[role='button'] span:contains('${link}')`).parent().click({ force: true });
+});
+And('user clicks on {string} to choose a resource to link', (selectContentLbl) => {
+    cy.get(`input[value='${selectContentLbl}']`).click({ force: true });
+})
+
+And('user selects {string} item from the list', (title) => {
+
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_content_browser').find(`td:contains(${title})`).parent().find('input').click({ force: true });
+});
+And('user clicks on {string} button to select item', (selectContent) => {
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_content_browser').find(`input[id='edit-submit'][value='${selectContent}']`).click({ force: true });
+});
+
+And('user fills out {string} field with {string}', (fieldLbl, value) => {
+    cy.get(`tbody label:contains("${fieldLbl}")`).parent().find('input').type(value);
+});
+
+And('user clicks on {string} button to link to a media', (link) => {
+    cy.get(`tbody summary[role='button'] span:contains('${link}')`).eq(1).parent().click({ force: true });
+});
+
+And('{string} had been selected', (title) => {
+    cy.get(`div:contains("${title}")`).should('be.visible');
+})
+
+And('user enters {string} into media title search box and clicks {string}', (nameToSearch, applyBtn) => {
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_media_browser').find('input#edit-name').type(nameToSearch);
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_media_browser').find('input#edit-submit-cgov-media-browser')
+        .click({ force: true });
+});
+And('user selects {string} item from the media list', (title) => {
+
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_media_browser').find(`td:contains(${title})`).parent().find('input').click({ force: true });
+});
+And('user clicks on {string} button to select media', (selectContent) => {
+    cy.getIframeBody('iframe#entity_browser_iframe_cgov_media_browser').find(`input[id='edit-submit'][value='${selectContent}']`).click({ force: true });
+});
+
```

Comment:
It's quite minor, but the formatting changes pattern here. It loses the white space between successive `And` functions and adds an unexpected white space in the middle one.  This makes no difference functionally, but it interferes with readability.

Reply:
addressed !